junit-platform-engine/README.md

```diff
@@ -246,7 +258,7 @@ Supported `DiscoverySelector`s are:
 The only supported `DiscoveryFilter` is the `PackageNameFilter` and only when
 features are selected from the classpath.
 
-### Selecting individual scenarios, rules and examples 
+### Selecting individual scenarios, rules and examples ###
 
 The `FileSelector` and `ClasspathResourceSelector` support a `FilePosition`.
```

Comment:
So they can be used to select a scenario on a specific line in the feature file? Where and how would I use these?

Reply:
You would do so by using JUnit 5 Launcher API. 

Or through dialog in IDEA configuration in Maven/ect once they support it.

https://junit.org/junit5/docs/current/user-guide/#advanced-topics

Reply:
Perhaps. I was thinking more along the lines of a short introduction to this paragraph, as this section seemed a bit out of left field for me. I've added a suggested intro; please suggest changes if I got it wrong.

Reply:
I was thinking more about what would be my use case to do so? Ie running the test I'm currently working on.